geonode_mapstore_client/client/js/components/DetailsPanel/DetailsLocations.jsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2023, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { useState, useEffect } from "react";
+import { connect } from 'react-redux';
+import turfCenter from "@turf/center";
+import isEmpty from "lodash/isEmpty";
+import isEqual from "lodash/isEqual";
+import get from "lodash/get";
+import wk from 'wellknown';
+
+
+import BaseMap from "@mapstore/framework/components/map/BaseMap";
+import mapTypeHOC from "@mapstore/framework/components/map/enhancers/mapType";
+import ZoomTo from "@js/components/ZoomTo/ZoomTo";
+import { getPolygonFromExtent, bboxToFeatureGeometry } from "@mapstore/framework/utils/CoordinatesUtils";
+import DrawSupport from "@js/components/DrawExtent";
+import { Message } from "@mapstore/framework/components/I18N/I18N";
+import tooltip from "@mapstore/framework/components/misc/enhancers/tooltip";
+import CopyToClipboardCmp from 'react-copy-to-clipboard';
+import Button from "@mapstore/framework/components/misc/Button";
+import FaIcon from "@js/components/FaIcon/FaIcon";
+import { setResourceExtent } from "@js/actions/gnresource";
+
+const Map = mapTypeHOC(BaseMap);
+Map.displayName = "Map";
+const CopyToClipboard = tooltip(CopyToClipboardCmp);
+
+const BoundingBoxAndCenter = ({extent, center, expand}) => {
```

Comment:
We don't need to expand the panel. The issue description shows that the info and the map should be always visible for all the resources

Reply:
Yes. Not needed when map should be visible for all resource